src/api/donors/donors.service.ts

```diff
@@ -26,6 +27,59 @@ export class DonorsService {
     return this.prisma.donor.findUnique({ where: { id } });
   }
 
+  async findAllByProjectId(projectId: number, args: PaginationArgs = { page: 1, itemsPerPage: 5, filter: '' }) {
+    const projectUsers = await this.prisma.projectUser.findMany({ where: { projectId } });
+    const userIds = projectUsers.map((projectUser) => projectUser.userId);
+
+    const projectDonors = await this.prisma.donor.findMany({
```

Comment:
Project Users and donors have nothing to do with each other. The project user are the members of the project, and the donors are those who made a economic donation.